File: Assignment 01/Program 14/prog14.c
//Write a Python program to get the number of occurrences of a specified element in an array.

#include<stdio.h>
#include<stdlib.h>

int main(){
    int n;
    printf("Enter the number of inputs: ");
    scanf("%d",&n);
    int *arr = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        printf("Enter the number %d: ",i+1);
        scanf("%d",(arr+i));
    }

    int ele, count = 0;
    printf("Enter the no. for counting occurences: ");
    scanf("%d",&ele);
    for (int i = 0; i < n; i++)
    {
        if(arr[i] == ele)
            count++;
    }

    printf("The no. of occurences of %d is %d.",ele,count);
    free(arr);
    
    return 0;
}

File: Assignment 01/Program 14/prog14.py
#Write a Python program to get the number of occurrences of a specified element in an array.

import array as arr

a1 = arr.array('i')

n = int(input("Enter the number of inputs: "))
for i in range(n):
    val = int(input(f"Enter the element {i+1}: "))
    a1.append(val)
count = 0
ele = int(input("Enter the number for counting occurences: "))
for j in range(n):
    if a1[j] == ele:
        count+=1

print(f"The no. of occurences of {ele} is {count}")